LiteCore/tests/VectorQueryTest.cc

```diff
@@ -227,23 +227,160 @@ N_WAY_TEST_CASE_METHOD(SIFTVectorQueryTest, "Hybrid Vector Query", "[Query][.Vec
         t.commit();
     }
     createVectorIndex();
+    QueryLanguage                         qlans[2] = {QueryLanguage::kJSON, QueryLanguage::kN1QL};
+    std::vector<std::pair<string, float>> result;
+    for ( unsigned lan = 0; lan < 2; ++lan ) {
+        QueryLanguage   queryLan = qlans[lan];
+        Retained<Query> query;
+        string          queryStr;
+        if ( queryLan == QueryLanguage::kJSON ) {
+            queryStr = R"(
+             ['SELECT', {
+                WHERE:    ['AND', ['VECTOR_MATCH()', 'vecIndex', ['$target']],
+                                  ['=', 0, ['%', ['._sequence'], 100]] ],
+                WHAT:     [ ['._id'], ['AS', ['VECTOR_DISTANCE()', 'vecIndex'], 'distance'] ],
+                ORDER_BY: [ ['.distance'] ],
+                LIMIT:    10
+             }] )";
+            queryStr = json5(queryStr);
+        } else if ( queryLan == QueryLanguage::kN1QL ) {
+            queryStr = "SELECT META().id, VECTOR_DISTANCE(vecIndex) AS distance FROM "s + collectionName
```

Comment:
I don't think we need to test everything with both query languages. If you want, you could have a separate test that both of those compile to the same SQL, without running anything.

Reply:
Okay, I will take it out to n1ql test.